### ISSUE TITLE
Get rid of @maxWidth and family

### ISSUE THREAD

**Issue by sandersaares** [Dash-Industry-Forum/DASH-IF-IOP#274]
v4 section 3.2.4 has wording along these lines:

> [...] the following attributes shall be present:
> @maxWidth (or @width if all Representations have the same width)

Plus similar for other "max" attributes.

These maxABC attributes seem to be a potential source of conflict and overall seem unnecessary. I have seen conflicting data in the real world and even created such content by accident!

I propose we remove the requirement and make an inverse recommendation: `@maxWidth` and family (all the other max) SHOULD NOT be present in an MPD. Clients are expected to calculate max values and SHALL NOT rely on `@maxWidth` and similar attributes.

This seems like a useful simplification to me.

PS. I looked at a few players and found zero usages of these "max" attributes. I think it is trivial for players to just iterate representations to find the max in any case.

**Comment by haudiobe** [comment]
I agree that max is not a good idea. I believe we should change the MAX values to be used for intended display setup. This is proposed in CTA WAVE.
